src/compiler/scanner/lines_1000_2000.rs

```diff
@@ -639,16 +1243,128 @@ impl Scanner {
                     return self.set_token(SyntaxKind::SemicolonToken);
                 }
                 CharacterCodes::less_than => {
+                    if is_conflict_marker_trivia(self.text(), self.pos()) {
+                        self.set_pos(scan_conflict_marker_trivia(
+                            self.text(),
+                            self.pos(),
+                            // Some(|diag, pos, len| self.error(on_error, diag, pos, len)),
```

Comment:
For some reason it was complaining about lifetimes with this as an `Option` so I just made it non-optional and passed a dummy callback in places that would've passed `None`